Smarter Private Construction/common/defines/zzz_mog_GSRD_def.txt

```diff
@@ -83,7 +83,8 @@ NAI = {
 	OWNER_BUILDING_LOCATION_NO_AVAILABLE_INFRASTRUCTURE_MULT = 0.1	# Multiply owner building location score by this if the state lacks infrastructure
 	OWNER_BUILDING_LOCATION_UNINCORPORATED_MULT = 0.01				# Multiply owner building location score by this if the state is unincorporated
 	
-	OWNER_COMPANY_SCORE_MULT = 30 # 3.0								# Multiply owner company score by this, the higher this is, the more likely a company building is chosen instead of other owning building types
+	OWNER_COMPANY_SCORE_MULT = 30 # 3.0								# Multiply owner company score by this in case of new construction, the higher this is, the more likely a company building is chosen instead of other owning building types
```

Comment:
```suggestion
	OWNER_COMPANY_SCORE_MULT = 5 # 3.0								# Multiply owner company score by this in case of new construction, the higher this is, the more likely a company building is chosen instead of other owning building types
```